api/v1/cluster_funcs.go

```diff
@@ -70,6 +70,30 @@ func (o OnlineConfiguration) GetImmediateCheckpoint() bool {
 	return *o.ImmediateCheckpoint
 }
 
+// GetRetryConfiguration returns the retry configuration for this volume snapshot
+// configuration. If no configuration is specified, returns nil.
+func (configuration *VolumeSnapshotConfiguration) GetRetryConfiguration() *VolumeSnapshotRetryConfiguration {
+	return configuration.RetryConfiguration
+}
+
+// GetDeadline returns the deadline for volume snapshot operations.
+// If no deadline is specified, returns the default value.
+func (c *VolumeSnapshotRetryConfiguration) GetDeadline() string {
+	if c == nil || c.Deadline == "" {
```

Comment:
If we validate this via the webhook, could we avoid checking this here? :thinking: